index.html

```diff
@@ -53,6 +54,241 @@ <h1>I'm Redwan.<br/>
+                                <li>
+                                    <a class="btn" href="">HTML</a>
+                                </li>
+                                <li>
+                                    <a class="btn" href="">CSS</a>
+                                </li>
+                                <li>
+                                    <a class="btn" href="">JavaScript</a>
+                                </li>
+                            </ul>
+                          </div>
+                          <div class="action">
+                            <button class="submit">See project</button>
+                          </div>
+                    </div>
+                    <div class="flex-col">
+                        <img
+                        class="snapshoot-portfolio-icon"
+                        alt=""
+                        src="Images/Snapshoot Portfolio (2).png"
+                      />
+                        <div class="top">
+                          <h2>Multi-Post Stories</h2>
+                          <div class="client-parent">
+                            <ul>
+                                <li>CANOPY</li>
+                                <li>Back End Dev </li>
+                                <li>2015</li>
+                            </ul>
+                          </div>
+                          <p>
+                            A daily selection of privately <br/>
+                            personalized reads; no accounts or <br/>
+                            sign-ups required.
+                          </p>     
+                        </div>
+                        <div class="tags">
+                            <ul>
+                                <li>
+                                    <a class="btn" href="">HTML</a>
+                                </li>
+                                <li>
+                                    <a class="btn" href="">CSS</a>
+                                </li>
+                                <li>
+                                    <a class="btn" href="">JavaScript</a>
+                                </li>
+                            </ul>
+                          </div>
+                          <div class="action">
+                            <button class="submit">See project</button>
+                        </div>
+                    </div>
+                </div>
+               </div>
+        </section>
+
+
+        <section class="section-bottom"> 
+            <div class="headerTop">
+                <h1>About Myself </h1>
+                <p >
+                    Hello I’m a software developer! I can 
+                    help you build a product , feature or 
+                    website. Look through some of my work 
+                    and experience! If you like what you see 
+                    and have a project you need coded, 
+                    don’t hesitate to contact me.
```

Comment:
- [ ] Your paragraph is not properly aligned 👽 Kindly fix that ❤️ 

![Screenshot from 2023-06-06 01-40-35](https://github.com/ethred/Portfolio/assets/10439283/8375a42d-c8c5-475f-adfd-040a7eb1be87)